### PR TITLE
Correct query

### PR DESCRIPTION
**Pull request by jmMeessen**
- all PRs in a repository with "hacltoberfest" topic count
- exclude PRs from bots

### CODE REVIEW

File: json_to_csv.jq [hunk 1]
```diff
@@ -4,13 +4,13 @@
 | map(
     select(
       # Spec: - Status is either open or merged
-      (.state == "open" or .pull_request.merged_at != null)
+      ((.state == "open" or .pull_request.merged_at != null) and (.user.type != "Bot"))
```
Comment (lemeurherve) [inline json_to_csv.jq:7]: 👏

**Review by lemeurherve — APPROVED** [review]

**Comment by NotMyFault** [comment]
Hey @jmMeessen,

once this PR is merged, I'm going ahead and take care of https://github.com/jenkins-infra/repository-permissions-updater/issues/2833, to prevent redoing this PR on the hosted repository.

Thanks!

**Comment by jmMeessen** [comment]
Hello @NotMyFault,

I thought to have time this WE to fix the last stretch (join the two queries). But instead enjoyed the nice autumn weather to do long walk with dog and family.

I will merge this PR this afternoon (Europe) and create a new one to refine the process.

**Comment by NotMyFault** [comment]
> I finally got around to merge this PR, @NotMyFault
> 
> You can go ahead to create the Hosted repo. Is something else required by me for that ?

Nope, you are good to go to archive the repository, once hosted on the infra org.

**Comment by NotMyFault** [comment]
All done, there we go 👏🏻

**Pull request merged**